packages/amplify_auth_cognito/android/src/test/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthCognitoPluginTest.kt

```diff
@@ -293,6 +293,28 @@ class AmplifyAuthCognitoPluginTest {
         verify(mockResult, times(1)).success(ArgumentMatchers.any<LinkedTreeMap<String, Any>>());
     }
 
+    @Test
+    fun getCurrentUser_returnsSuccess() {
+        // Arrange
+        doAnswer { invocation: InvocationOnMock ->
+            plugin.prepareUpdatePasswordResult(mockResult)
+            null as Void?
+        }.`when`(mockAuth).getCurrentUser()
+
+        val data: HashMap<*, *> = hashMapOf(
```

Comment:
Does it need to be a mutable map? And does it need to be a mutable map of the hash variety? Why not a default immutable
```suggestion
        val data: Map<*, *> = mapOf(
```

Related: [a discussion on `hashMapOf` vs `mutableMapOf`](https://stackoverflow.com/a/51563494/695787)

Reply:
I think this could a Map as you say, but I'm going to leave this for a larger refactor of the tests.  I need to look at the Flutter method channel before making this change.